spec/system/index_spec.rb

```diff
@@ -67,18 +65,17 @@
     end
     it 'displays a helpful message' do
       expect(page).to have_selector 'p.notice',
-                                    text: 'Stop 1234 not found'
+                                    text: "Stop #{incorrect_stop_name} not found"
     end
   end
   context 'without completing stop name' do
-    let!(:bus_stop) { create :bus_stop, name: 'Mill Valley Apartments' }
     it 'autofills' do
-      fill_in 'Enter stop name', with: 'Mill Valley'
-      find('div', text: 'Mill Valley Apartments').click
+      fill_in 'Enter stop name', with: bus_stop.name.chars.first
```

Comment:
I'm confused, why the first character?

Reply:
Because I want to test that autofill works. In order to do that, I fill in the `text_field` with the first letter of the stop name: `M`. Then `autocomplete` will create a `div` with `Mill Valley Apartments`.

Reply:
Okay, I see. Perhaps a comment to that effect might be useful - or, making the reasonable decision that most people who read this will be smarter than Dave.

Reply:
I'll write in a comment.